.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: tests
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: "7 7 * * *"
```

Comment:
Similar comment here: What's the value of running our tests regularly? We are using Go Modules which pins our dependencies (afaik) and we have our Go version pinned so unless Go introduces a breaking change why would we need to run our tests on a schedule?